laCour/slack-night-mode
# add texty_legacy selector for ql-placeholder text

This corrects the color of the placeholder text in the message box by
adding an additional selector to the style override we already have.

Test Plan:
- Visit any channel or DM
- Verify that the placeholder text in the message box ("Message {channel_name}")
  is a light grey color, not black
